tox.ini

```diff
@@ -4,7 +4,7 @@ skipsdist = True
 skip_missing_interpreters = True
 isolated_build = True
 toxworkdir = {env:TOXWORKDIR:/tmp/{env:USER}/tox}
-envlist = lint,server,agent,py3-server
+envlist = lint,agent,py3-server,server
```

Comment:
What is the relationship between this definition and lines 30 and 40?  (If we are removing `server` and `agent` from here, should we be removing the `[testenv:server]` and `[testenv:agent]` sections below?)

Reply:
The order of the `envlist` determines the order in which they are executed without the `-e` switch.  They are not tied to the order of the sections in the file.